views/footer.php

```diff
@@ -1,5 +1,61 @@
 	</main>
   <!-- Site content ends -->
+  <?php 
+    if (isset($messageNum) || isset($_GET['m'])) {
+      if (!isset($messageNum) && isset($_GET['m'])) {
+        $messageNum = $_GET['m'];
+      }
+      if ($messageNum < 50) {
+        $notifyClass = "bg-success";
+      } else {
+        $notifyClass = "bg-danger";
+      }
```

Comment:
Checks the message number. If it's under 50, it's a "success" message and the background is tinted green. If it's 50 or over, it's a "failure" message and the background is tinted red.